src/components/DropMenu/DropMenu.reducers.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
+
+export function onExpand(state) {
+	const { selectedIndices } = state;
+
+	return _.assign({}, state, {
```

Comment:
Use splats instead of assign:

```
return {
  ...state,
  isExpanded: true,
  focusedIndex: _.isEmpty(selectedIndices) ? null : _.last(selectedIndices)
}
```